### PR TITLE
[Staging] Share indicator on twitter

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Convert the indicators to images that can be viewed on twitter cards.

To test:

You will have to use ngrok to expose your localhost and then use this validator https://cards-dev.twitter.com/validator

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

screen_shot_2019-05-17_at_4 45 36_pm

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
@nerdlet @kilemensi @KhadijaMahanga @phillipahereza 

Hey guys so it seems twitter won’t allow more that one query parameter in their share example `https://twitter.com/intent/tweet?url=http://localhost:3000/profiles/senegal?one=1&two=2` only param `one` is taken. I need to pass 3 `?indicator=&title=&description=`. Any ideas for a simple url shortener I can use or do you think I should install a link shortener django app.

**Comment by karimkawambwa** [comment]
Screen Shot 2019-05-23 at 5 21 20 PM

@phillipahereza lets make sure we always have one image.

**Comment by karimkawambwa** [comment]
@kilemensi ready

**Comment by karimkawambwa** [comment]
@kilemensi @phillipahereza

File: takwimu/templates/takwimu/_layouts/_base.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% load static sass_tags meta jsonify wagtailcore_tags %}
+{% load staticfiles sass_tags meta jsonify %}
```
[the remark this answers is not in the document]
Comment (kilemensi) [reply takwimu/templates/takwimu/_layouts/_base.html]: Put another way does `static` handle `S3` storage @phillipahereza or do we still need `staticfiles`?

File: takwimu/templates/takwimu/_layouts/_base.html [hunk 2]
```diff
@@ -56,10 +57,6 @@
     takwimu.url = '{{ HURUMAP.url }}';
     takwimu.countries = {{ countries|jsonify|safe }};
     takwimu.settings = {
-      navigation: {
-        country_analysis: '{{ settings.takwimu.NavigationSetting.country_analysis|richtext }}',
-        data_by_topic: '{{ settings.takwimu.NavigationSetting.data_by_topic|richtext }}'
-      },
```
[the remark this answers is not in the document]
Comment (kilemensi) [reply takwimu/templates/takwimu/_layouts/_base.html]: We also need `wagtailcore_tags` loaded for the `richtext` filter.

[the remark this answers is not in the document]
Comment (karimkawambwa) [reply takwimu/templates/takwimu/_layouts/_base.html]: probably bad merge

File: takwimu/templates/takwimu/_layouts/_base.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (karimkawambwa) [reply takwimu/templates/takwimu/_layouts/_base.html]: this is probably merge issue, i will return to what is in staging

**Comment by karimkawambwa** [comment]
> Shouldn't share only be available where site visitor can download image, and

I'll disable html sharing until further notice, Its currently mixed with iframes 

> Does Twitter handle images of all sizes and shape or we need to make sure they fit certain sizes?

Handled, I use rendition and set the size required.
@kilemensi

**Comment by karimkawambwa** [comment]
@kilemensi 

Review one more time

**Pull request merged**